modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -417,20 +425,20 @@ definition {
 			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject170529");
 
 			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
-				fieldName = "Custom Field 170529",
+				fieldName = "customObjectField",
 				objectName = "CustomObject170529",
 				value = "Entry Test");
-
-			JSONAccountEntry.addAccountEntry(
-				accountEntryName = "Account Name 1",
-				accountEntryType = "Business");
-
-			JSONAccountEntry.addAccountEntry(
-				accountEntryName = "Account Name 2",
-				accountEntryType = "Business");
 		}
 
 		task ("When the user creates a filter and the relationship column can be selected as the filter") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject170529");
+
+			ObjectPortlet.viewEntryDetails(entry = "Entry Test");
+
+			ObjectPortlet.searchEntryRelationshipField(entryValue = "Account Name 1");
+
+			PortletEntry.save();
+
```

Comment:
Why are we navigating to the object entry and then searching for the account entry?

Reply:
Hey @timpak , I use this macro becuase the macro **ObjectAdmin.addObjectSingleFieldEntryViaAPI** do not made the entry on relationship field. Do you have another option the make the entry on relationship field on this test?